azerothcore/azerothcore-wotlk
# fix(DB/Loot): Revenge of Gann (Part 1) Quest Drop Rate too high

## Changes Proposed:
-  Drop Rate now

| Item  | Bael'dun Soldier | Bael'Dun Rifleman  | Bael'Dun Officer | Lord Cyrik Blackforge |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| Nitroglycerin  | 28%  | 29%  | 28%  | 7%  |
| Sodium Nitrate  | 28%  | 29%  | 27%  | 7%  |
| Wood Pulp  | 28%  | 29%  | 28%  | 7%  |

## Issues Addressed:
- Closes #6650 

## SOURCE:
- https://classic.wowhead.com/quest=846/revenge-of-gann
- https://classic.wowhead.com/item=5017/nitroglycerin#dropped-by
- https://classic.wowhead.com/item=5017/nitroglycerin#dropped-by
- https://classic.wowhead.com/item=5019/sodium-nitrate

## Tests Performed:
-  Tested on local server

## How to Test the Changes:


1. Take Quest Revenge of Gann Part 1
2. .go creature 13649
3. Check drop rate
4.  Or check in Keira3



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
